crates/next-core/src/nodejs/pool.rs

```diff
@@ -52,7 +52,7 @@ impl Drop for RunningNodeJsPoolProcess {
     }
 }
 
-const CONNECT_TIMEOUT: Duration = Duration::from_secs(1);
+const CONNECT_TIMEOUT: Duration = Duration::from_millis(10);
```

Comment:
Wait, isn't this reducing the timeout from 1s to 10ms? Your PR description says we should be increasing it.

Reply:
Whoops, forgot to change it back. Good catch.